gazebo_plugins/src/gazebo_ros_wheel_slip.cpp

```diff
@@ -19,10 +19,17 @@
 #include <gazebo_plugins/gazebo_ros_wheel_slip.hpp>
 #include <gazebo_ros/node.hpp>
 #include <std_msgs/msg/bool.hpp>
+#include <rclcpp/rclcpp.hpp>
+#include "gazebo_msgs/msg/instant_slip.hpp"
 
 #include <memory>
 #include <string>
 #include <vector>
+#include <cmath>
+
+#define PI 3.14159265
```

Comment:
use `M_PI` from the standard headers or `IGN_PI` from `ignition/math/Helpers.hh`

Reply:
Removed in https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1331/commits/fdeefa9cbd3b46447921e4b24df66fb59ce603c6.